akka-operator/values.yaml

```diff
@@ -42,6 +42,22 @@ resources:
     cpu: 500m
     memory: 512Mi
 
+readinessProbe:
+  httpGet:
+    port: 8558
+    path: "/ready"
+  periodSeconds: 10
+  initialDelaySeconds: 20
+  failureThreshold: 10
+
+livenessProbe:
+  httpGet:
+    port: 8558
+    path: "/alive"
+  periodSeconds: 10
+  initialDelaySeconds: 20
+  failureThreshold: 10
```

Comment:
I think we can have less threshold here. This means that we wouldn't restart until after 10x10 = 100 seconds?
Maybe 5 ?

Reply:
you are right, even lower, I went for 3 !